tools/airgap_linter.py

```diff
@@ -26,7 +26,7 @@ def readlines_if_text_file(filename):
 def extract_uris(file_name):
     lines = readlines_if_text_file(file_name)
 
-    matcher = re.compile(r".*https?:\/\/([^\?\s]*)", re.IGNORECASE)
+    matcher = re.compile(r".*http?:\/\/([^\?\s]*)", re.IGNORECASE)
```

Comment:
Just fyi, this may break downstream packages (edgelb, dss, k8 etc who use our tooling), but I can't think of a reason that breakage is not easily fixable.

Reply:
I agree to this, but this was the purpose of the whole ticket to remove all non-HTTPS assets. I hope that this will us tackle those resources well.